vendor/go.etcd.io/etcd/client/v3/watch.go

```diff
@@ -580,6 +589,26 @@ func (w *watchGrpcStream) run() {
 
 			switch {
 			case pbresp.Created:
+				cancelReasonError := v3rpc.Error(errors.New(pbresp.CancelReason))
```

Comment:
still waiting on a revert of in the 3.5 stream before pulling in an etcd update

xref https://github.com/kubernetes/kubernetes/pull/114403#discussion_r1045906333

Reply:
/hold

Reply:
also https://github.com/kubernetes/kubernetes/pull/114403 :)